packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -0,0 +1,195 @@
+    fill: currentcolor;
+    height: 16px;
+    width: 16px;
+    border: ${strokeWidthThin} solid ${colorNeutralStrokeAccessible};
+    border-radius: ${borderRadiusSmall};
+    outline: none;
+  }
+  .label {
+    align-self: center;
+    cursor: inherit;
+    padding-inline: ${spacingHorizontalS};
+    padding-bottom: ${spacingVerticalS};
+    padding-top: ${spacingVerticalS};
+  }
+  .label__hidden {
+    display: none;
+    visibility: hidden;
+  }
+  .checked-indicator {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: ${colorNeutralForegroundInverted};
+    font-size: 12px;
+    margin: auto;
+    opacity: 0;
+  }
+  .indeterminate-indicator {
+    position: absolute;
+    width: 8px;
+    height: 8px;
+    border-radius: ${borderRadiusSmall};
+    background-color: ${colorCompoundBrandForeground1};
+    opacity: 0;
+  }
+  :host(:hover) {
+    color: ${colorNeutralForeground2};
+  }
+  :host(:hover) .control {
+    border-color: ${colorNeutralStrokeAccessibleHover};
+  }
+  :host(:hover) .indeterminate-indicator {
+    background-color: ${colorCompoundBrandForeground1Hover};
+  }
+  :host(:active) .control {
+    border-color: ${colorNeutralStrokeAccessiblePressed};
+  }
+  :host(:active) .indeterminate-indicator {
+    background-color: ${colorCompoundBrandForeground1Pressed};
+  }
+  :host([aria-checked='true']),
+  :host([aria-checked='mixed']),
+  :host(:active) {
+    color: ${colorNeutralForeground1};
+  }
+  :host([aria-checked='true']) .control {
+    background-color: ${colorCompoundBrandBackground};
+  }
+  :host([aria-checked='true']:hover) .control {
+    border-color: ${colorCompoundBrandStrokeHover};
+    background-color: ${colorCompoundBrandBackgroundHover};
+  }
+  :host([aria-checked='true']:active) .control {
+    background-color: ${colorCompoundBrandBackgroundPressed};
+  }
+  :host([aria-checked='mixed']:hover) .control {
+    border-color: ${colorCompoundBrandStrokeHover};
+  }
+  :host([aria-checked='true']) .checked-indicator,
+  :host([aria-checked='mixed']) .indeterminate-indicator {
+    opacity: 1;
+  }
+  :host([aria-checked='true']) .control,
+  :host([aria-checked='mixed']) .control {
+    border-color: ${colorCompoundBrandStroke};
+  }
+  :host([aria-checked='mixed']:active) .control,
+  :host([aria-checked='true']:active) .control {
+    border-color: ${colorCompoundBrandStrokePressed};
+  }
+  :host([label-position='before']) {
+    flex-direction: row-reverse;
+  }
+  :host([label-position='before']) .label {
+    padding-inline: ${spacingHorizontalS} ${spacingHorizontalXS};
+    padding-top: ${spacingVerticalS};
+    padding-bottom: ${spacingVerticalS};
+  }
+  :host([size='large']) .control {
+    width: 20px;
+    height: 20px;
+  }
+  :host([size='large']) .checked-indicator {
+    font-size: 16px;
+    height: 20px;
+  }
+  :host([aria-checked='mixed'][size='large']) .indeterminate-indicator {
+    width: 10px;
```

Comment:
The size of the large indeterminate indicator is now 10x8 (for both square and circular), I guess it should be 10x10:
<img width="274" alt="image" src="https://github.com/microsoft/fluentui/assets/9615899/a254504b-8df3-4cf2-99d2-d8a0c8b9437e">